docker/docker.go

```diff
@@ -67,6 +67,11 @@ func main() {
 		flHosts = append(flHosts, defaultHost)
 	}
 
+	file := "/etc/docker/key.json"
+	info, _ := os.Stat(file)
```

Comment:
In the future, please don't ignore errors: this resulted in a panic as you can see on jenkins: https://jenkins.dockerproject.com/job/Docker-PRs/3679/console

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x30 pc=0x4094f3]

goroutine 1 [running]:
main.main()
	/go/src/github.com/docker/docker/docker/docker.go:72 +0x4d3

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 8 [runnable]:
text/template/parse.lexText(0xc208054180, 0xea5ea0)
	/usr/local/go/src/text/template/parse/lex.go:228 +0x37b
text/template/parse.(*lexer).run(0xc208054180)
	/usr/local/go/src/text/template/parse/lex.go:198 +0x5d
created by text/template/parse.lex
	/usr/local/go/src/text/template/parse/lex.go:191 +0x1ac
```